test/runtests.jl

```diff
@@ -738,6 +738,11 @@ HISTORY this is a history"""
             # Test that show() works and that the beginning of output is what we expect.
             @test repr(f)[1:6] == "File: "
 
+            # Test the deletion of a key
+            dhdr = deepcopy(inhdr)
+            delete!(dhdr, "FLTKEY")
+            @test !haskey(dhdr, "FLTKEY")
+
```

Comment:
```suggestion
@test_throws KeyError delete!(dhdr, "aaabbbbccccdddd")
```
make sure we throw an error, as expected

Reply:
Implemented - all good suggestions, thanks